sources/executor/builtins/builtins.c

```diff
@@ -0,0 +1,55 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        ::::::::            */
+/*   builtins.c                                         :+:    :+:            */
+/*                                                     +:+                    */
+/*   By: qtrinh <qtrinh@student.codam.nl>             +#+                     */
+/*                                                   +#+                      */
+/*   Created: 2024/03/01 14:47:56 by qtrinh        #+#    #+#                 */
+/*   Updated: 2024/03/01 20:05:28 by qtrinh        ########   odam.nl         */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../../../includes/minishell.h"
+
+bool	is_builtin(char *cmd)
+{
+	if (ft_strncmp(cmd, "echo", 5) == 0)
```

Comment:
I think we should use a jump table here. `is_builtin` should return an enum value. Which will be passed to the jump table.